app/design/frontend/Magebit/learning/Magento_Catalog/templates/product/view/gallery.phtml

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+/**
+ * Product media data template
+ *
+ * @var $block \Magento\Catalog\Block\Product\View\Gallery
```

Comment:
Use istead this type of declaration

/** @var Class $classObject */
/** @var Class1 $classObject1 */
/** @var Class2 $classObject2 */